addons/mass_mailing/static/src/xml/mass_mailing_views.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
```

Comment:
Maybe name it ``mailing_contact_views`` to be more precise on the underlying model ?